example/src/components/ButtonContainer/ButtonContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './ButtonContainer.css'
+
+/**
+ * One custom component from a application that uses Nice!, like Search!
```

Comment:
Maybe change the English a bit: An example of how to use Nice based styling in a component implemented in a 3rd party application, such as Search!